### PR TITLE
updated finance runbook

### PR DESCRIPTION
**Pull request by moutons**

### CODE REVIEW

File: finance_runbook.md [hunk 1]
```diff
@@ -6,50 +6,22 @@ Generally, this is a core role.
...
-
-### The Day Before
+### General Tasks
 
-* Set up.
-* Party.
+* Establish budget (Core)
+* Maintain budget (Everyone)
+* Submit expenses (Them what spent it, via email, add the receipt in the year's Receipts folder)
+* Reimburse expenses (Core, ConferenceOps)
```
Comment (nimbinatus) [inline finance_runbook.md:19]: I'd still have the important dates in here:

- Budget set 10 mo out (or pre-kickoff). <-- necessary for sponsor prospectus, venue choice.
- Budget check-ins would be good throughout the season, like maybe before tickets open, 6 mo out, when early bird closes, 3 mo out...
- Books close date (I'd check with Laura W on this one, but I think it's June?)

**Review by nimbinatus — CHANGES_REQUESTED** [review]
Minor note on timelines

File: finance_runbook.md (hunk 1 above)
Comment (nimbinatus) [reply finance_runbook.md:19]: @moutons ^^ thoughts?

Comment (moutons) [reply finance_runbook.md:24]: see the recent updates

**Review by nimbinatus — APPROVED** [review]

**Review by nimbinatus — APPROVED** [review]

**Pull request merged**